docker-compose/base/bundle-va/README.md

```
@@ -0,0 +1,321 @@
+
+#### Configuration <div id='model_download_conf'>
+
+Not needed for model_download
+
+#### Credentials <div id='model_download_creds'>
+
+To enable model download, you need to provide credentials for the bucket where models are stored. 
+Google Cloud and S3 buckets are supported.
+
+If your models are stored on gcs, the [service account](https://cloud.google.com/iam/docs/service-accounts-create) credentials must be in /<deployment_dir>/creds/bundle/gcs.json file: 
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)